presto-docs/src/main/sphinx/admin/properties.rst

```diff
@@ -82,48 +83,58 @@ Properties controlling spilling
 ``experimental.spiller-spill-path``
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-    * **Type:** ``String``
+    * **Type:** ``string``
     * **No default value.** Must be set when spilling is enabled
-    * **Description:** Directory where spilled content will be written. It can be a comma separated list to
 
-    spill simultaneously to multiple directories, which helps to utilize multiple drives installed in the system.
-    It is not recommended to spill to system drives. Especially do not spill on a drive, to which are
-    written JVM logs, as disk overutilization might cause JVM to pause for lengthy periods causing queries to fail.
+    Directory where spilled content will be written. It can be a comma separated
+    list to spill simultaneously to multiple directories, which helps to utilize
+    multiple drives installed in the system.
+
+    It is not recommended to spill to system drives. Most importantly, do not spill
+    to the drive on which the JVM logs are written, as disk overutilization might
+    cause JVM to pause for lengthy periods, causing queries to fail.
 
 ``experimental.spiller-minimum-free-space-threshold``
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
- * **Type:** ``Double``
- * **Default value:** ``0.9``
- * **Description:** If disk space usage of a given spill path is above this threshold, this spill path will not be eligible for spilling.
+    * **Type:** ``double``
+    * **Default value:** ``0.9``
+
+    If disk space usage ratio of a given spill path is above this threshold,
+    this spill path will not be eligible for spilling.
 
 ``experimental.spiller-threads``
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
- * **Type:** ``Integer``
- * **Default value:** ``4``
- * **Description:** Number of spiller threads. Increase this value if the default is not able to saturate the underlying spilling device (for example, when using a RAID matrix with multiple disks)
+    * **Type:** ``integer``
+    * **Default value:** ``4``
+
+    Number of spiller threads. Increase this value if the default is not able
+    to saturate the underlying spilling device (for example, when using RAID).
```

Comment:
why not `disk(s)` instead of `spilling device`?  spilling device doesn't mean much.

Reply:
I'll also replace "drives" with "disks" to be consistent.

Reply:
Actually, "device" is more accurate here, as that matches UNIX terminology, and is more generic so it works for SSD, NVMe, etc.